sequencer/api/migrations/postgres/V501__epoch_tables.sql

```diff
@@ -17,7 +17,6 @@ CREATE TABLE vid_share2 (
     data BYTEA
 );
 
-
```

Comment:
Unfortunately even this blank line diff might cause an issue. I believe the migration runner does a simple text comparison with the deployed migrations before applying new ones to make sure it's not doing anything inconsistent